build_system/docker_build.py

```diff
@@ -0,0 +1,77 @@
+
+import re
+import subprocess
+import sys
+
+from cross_build import BuildSystem
+import constants as c
+import build_utils as utils
+
+class DockerBuildSystem(BuildSystem):
+    def clean(self, config):
+        try:
+            self.exec_host_cmd("docker rm -f -v j2v8.$PLATFORM.$ARCH", config)
+        except subprocess.CalledProcessError:
+            return
+
+    def health_check(self, config):
+        try:
+            # general docker availability check
+            self.exec_host_cmd("docker stats --no-stream", config)
+
+            # NOTE: the additional newlines are important for the regex matching
+            version_str = utils.execute_to_str("docker version") + "\n\n"
```

Comment:
This did the trick, can you add this somewhere to pyton script or git repo that that those scripts have to be executable?

Now building is going further